content/id/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -0,0 +1,281 @@
+
+Nama setiap Container aplikasi dan Init Container pada sebuah Pod haruslah unik; Kesalahan validasi akan terjadi jika ada Container atau Init Container yang memiliki nama yang sama.
+
+### Sumber Daya
+
+Karena eksekusi Init Container yang berurutan, aturan-aturan untuk sumber daya berlaku sebagai berikut:
+
+* Yang tertinggi antara `request` atau `limit` sumber daya yang didefinisikan pada **semua Init Container** adalah **`request`/`limit` inisialisasi yang berlaku**.
+* _`request`/`limit`_ sumber daya Pod yang berlaku adalah yang paling besar diantara:
+  * Jumah `request`/`limit` semua Container aplikasi untuk suatu sumber daya.
+  * `request`/`limit` inisialisasi yang berlaku untuk suatu sumber daya.
+* Penjadwalan dilakukan berdasarkan `request`/`limit` (Pod) yang berlaku, yang berarti bahwa Init Container dapat mengambil sumber daya inisialisasi yang tidak digunakan selama umur Pod tersebut.
+* _Tingkat QoS yang berlaku_ milik Pod adalah sama dengan tingkat QoS untuk Init Container dan Container aplikasi.
+
+`ResourceQuota` dan `limitedResources` diberlakukan berdasarkan `request` dan `limit` Pod yang berlaku.
+
+Cgroup pada tingat Pod didasarkan pada `request` dan `limit` Pod yang berlaku, sama dengan _scheduler_.
+
+### Alasan Pod diulang kembali
+
+Pod dapat diulang kembali, yang berakibat pada diulangnya eksekusi Init Container, diakibatkan oleh beberapa alasan berikut:
+
+* Seorang pengguna memperbarui `PodSpec`, mengakibatkan `image` Init Container berubah. Perubahan apapun pada `image` Init Container akan mengulang kembali Pod tersebut. Perubahan pada `image` Container aplikasi hanya mengulang kembali Container aplikasi yang bersangkutan.
+* Infrastruktur Container Pod diulang kembali. Hal ini jarang terjadi, dan hanya dapat dilakukan oleh seseorang yang memiliki akses _root_ pada _node_ yang bersangkutan.
+* Semua Container di dalam Pod diterminasi, dengan nilai `restartPolicy` yang disetel sebagai `Always`, memaksa pengulangan kembali, dan catatan selesainya Init Container telah hilang karena _garbage collection_.
+
+## Dukungan dan kompatibilitas
+
+Sebuah kluster dengan versi Apiserver 1.6.0 keatas mendukung Init Container melalui kolom `.spec.initContainers`. Versi-versi sebelumnya mendukung Init Container melalui anotasi _alpha_ atau _beta_. Kolom `.spec.initContainers` juga diduplikasikan dalam bentuk anotasi `alpha` dan `beta` agar Kubelet versi 1.3.0 keatas dapat menjalankan Init Container, dan agar Apiserver versi 1.6 dapat dengan aman dikembalikan ke versi 1.5.x tanpa kehilangan fungsionalitas Pod-pod yang telah dibuat sebelumnya.
+
+Pada Apiserver dan Kubelet versi 1.8.0 keatas, dukungan untuk anotasi `alpha` dan `beta` telah dihapus, sehingga dibutuhkan konversi (manual) dari anotasi yang telah kedaluarsa tersebut ke dalam bentuk kolom `.spec.initContainers`.
```

Comment:
I think alpha beta doesn't need we should make alpha/beta format consistent (with italic? like the rest)
```suggestion
Pada Apiserver dan Kubelet versi 1.8.0 ke atas, dukungan untuk anotasi _alpha_ dan _beta_ telah dihapus, sehingga dibutuhkan konversi (manual) dari anotasi yang telah kedaluwarsa tersebut ke dalam bentuk kolom `.spec.initContainers`.
```